### ISSUE TITLE
Проблем с CSS на нова инсталация

### ISSUE THREAD

**Issue by basekat** [chitanka/chitanka#86]
Прясно инсталиран Debian bullseye и инсталация с [Автоматичния инсталатор](https://github.com/chitanka/chitanka-installer). Моята библиотека изглежда както е показано на скрийншота по-долу:

![image](https://user-images.githubusercontent.com/16616757/173158303-b11bf6cf-9049-48b8-9a85-2fc7dfee53eb.png)

Проблемът е в следния URL: /css/?skin=orange&menu=right
![image](https://user-images.githubusercontent.com/16616757/173158617-5083f5c8-74fc-465a-a401-d6cff2c03756.png)

Същият проблем е описан тук:
https://forum.chitanka.info/post28047.html

Следвах инструкциите описани в поста, но нямаше резултат.

**Comment by bmanolov** [comment]
Версията на Less.php е прекалено стара и не е съвместима с версията на PHP, която ползваш. Предполагам, че си с PHP 7.4.

След няколко дена ще опитам да сложа по-нова версия.

Като цяло, целия софтуер трябва да се обнови, защото и други части не работят добре върху PHP 7.4.

**Comment by basekat** [comment]
Да, в случая беше 7.4.

Понеже пробвам всичко това в Docker, беше лесно да сваля и пробвам с php-7.3-buster. За съжаление същия резултат:
![image](https://user-images.githubusercontent.com/16616757/174035901-4e9ee714-e4b7-4804-814c-a596c883d224.png)

Прикачам и phpinfo() информация:
[PHP 7.3.33 - phpinfo().html.gz](https://github.com/chitanka/chitanka/files/8917510/PHP.7.3.33.-.phpinfo.html.gz)

**Comment by bmanolov** [comment]
Да, пропуснах да спомена, че този вид проверка/грешка се появява при PHP 7.3. :-)

**Comment by basekat** [comment]
Потвърждавам, че работи с PHP 7.2 :) Благодаря за  информацията.

Ще чакам с нетърпение обновлението на софтуера.

**Comment by bmanolov** [comment]
Пакета за LESS е обновен и би трябвало да работи с последните версии на PHP.

Актуализирането на останалите пакети ще се позабави, защото се налага преработка на цялостния софтуер, за да работи с последните версии на Symfony и Doctrine.

**Comment by basekat** [comment]
Благодаря за информацията. Ще тествам и докладвам.

**Comment by basekat** [comment]
Тествах успешно. Благодаря! 
Открих и проблем описан в #90 .
